app/src/main/java/es/wolfi/app/passman/CredentialEdit.java

```diff
@@ -240,17 +297,33 @@ public void onClick(View view) {
                     public void onSuccess(int statusCode, cz.msebera.android.httpclient.Header[] headers, byte[] responseBody) {
                         String result = new String(responseBody);
                         if (statusCode == 200 && !result.equals("")) {
-                            Snackbar.make(view, R.string.successfully_deleted, Snackbar.LENGTH_LONG)
-                                    .setAction("Action", null).show();
-                            assert getFragmentManager() != null;
-                            alreadySaving = false;
+                            try {
```

Comment:
This code seems to be repeated a few times, wonuldn't it be better if it was moved to it's own function?
I'm not sure if it might make sense to move it to the api library or if it's too specific for that.

Reply:
replaced with one of the new response handlers